index.html

```diff
@@ -79,14 +82,13 @@ <h3 style="color:#FF7F00">Take advantage of these tools to help secure your envi
         <!-- Example row of columns -->
         <div class="row">
           <div class="col-md-8 col-md-offset-2 grow" style="border: 10px solid #497CE2;">
-            <h2><i class="fa fa-cogs"></i> Property Management</h2>
-            <h3 style="color:#497CE2">Alter your configuration without restarting or redeploying.</h3>
-            <p class="cat-desc">In this section, you'll find both our small-scale, easy-to-use centralized configuration tool <a href="https://github.com/rapid7/conqueso">conqueso</a> along with its various clients.  In addition to that, we have also included our next-gen, open source project <a href="https://github.com/rapid7/propsd">propsd</a> which combines the power of S3 along with <a href="https://www.consul.io/">Consul.io</a> to provide a drop in replacement for <a href="https://github.com/rapid7/conqueso">conqueso</a>.</p>
+            <h2><i class="fa fa-cogs"></i> Configuration Management</h2>
+            <h3 style="color:#497CE2">Alter and secure your configuration without restarting or redeploying.</h3>
+            <p class="cat-desc">In this section, you'll find our easy-to-use centralized configuration management tools like <a href="https://github.com/rapid7/propsd">propsd</a> which combines the power of S3 along with <a href="https://www.consul.io/">Consul.io</a> to deliver dynamic configuration and catalog changes.  We are also including here our tooling for encrypting secrets via <a href="https://github.com/rapid7/tokend">tokend</a> and <a href="https://github.com/rapid7/warden">warden</a>.</p>
```

Comment:
Do we want to capitalize Propsd, Tokend, and Warden here? Also, I think it's just "Consul" and not "Consul.io".